src/pages/guide/integration/index.md

```diff
@@ -0,0 +1,377 @@
+---
+
+# Integration testing
+
+Integration tests require the Commerce runtime environment, so they need a little preparation before they can be executed.
+
+After preparing the system, you can execute tests using the command line interface or within an IDE like PhpStorm.
+
+## Set up the integration test framework
+
+To run integration tests, you must create and configure a test database.
+You may also want to adjust the PHPUnit configuration, depending on your requirements.
+
+See [Preparing Integration Test Execution](#prepare-integration-test-execution).
+
+## Command Line Interface (CLI)
+
+This option can be used for running tests locally during development or on remote servers during continuous integration.
+
+See [Running Integration Tests in the CLI](#run-integration-tests-in-the-cli).
+
+## PhpStorm IDE
+
+Running integration tests inside an IDE, like PhpStorm, is convenient during development. This is mostly used when writing a new integration test.
+
+Other than convenience, there is no benefit over running the tests on the console.
+
+See [Running Integration Tests in PhpStorm][phpstorm run].
+
+## Prepare integration test execution
+
+Before you can use the integration test framework, you must prepare the test environment.
+Prerequisites for the test environment include the following:
+
+-  A dedicated integration test database
+-  The test framework database configuration
+-  The PHPUnit configuration matches the purpose of the integration test execution
+
+## Integration test database
+
+By default, for every integration test run, the test framework installs a fresh test database.
+
+<InlineAlert variant="warning" slots="text" />
+
+Do not use the same database as the real Commerce instance. Any data, such as products, customers, orders, and everything else, will be lost!
+
+For safety reasons, it is recommended to use a dedicated database user for running the tests. That database user should not have access to any other databases.
+Here are examples of SQL commands that create a test database and a dedicated test user account:
+
+```sql
+CREATE DATABASE magento_integration_tests;
+GRANT ALL ON magento_integration_tests.* TO 'magento2_test_user'@'localhost' IDENTIFIED BY 'ftYx4pm6^x9.&^hB';
+```
+
+Replace the example database, username, and password with something that matches your requirements and conventions.
+
+### Configure the framework for test environment
+
+The integration test framework comes with a configuration file template
+`dev/tests/integration/etc/install-config-mysql.php.dist`.
+
+Copy this file to
+`dev/tests/integration/etc/install-config-mysql.php`
+(without the `.dist` suffix) and add your test database access credentials.
+
+The contents will look similar to the following. Each array key will be passed as an option argument when the test
+framework executes the `setup:install` command. Use the database access credentials for your test database instead of the
+example values below.
+
+```php
+<?php
```

Comment:
Actually, those are [opening tags](https://www.php.net/manual/en/language.basic-syntax.phptags.php) that indicate the beginning of a PHP file. I think they should stay since it's valid PHP code.

Reply:
i see, i must have misinterpreted [this comment](https://magento.slack.com/archives/C02L797RK5H/p1658183564679449?thread_ts=1658177638.948459&cid=C02L797RK5H).